python/ray/data/tests/test_hudi.py

```diff
@@ -0,0 +1,98 @@
+import os
+import zipfile
+
+import pyarrow as pa
+import pytest
+from pytest_lazyfixture import lazy_fixture
+
+import ray
+from ray.data.datasource.path_util import _unwrap_protocol
+
+PYARROW_LE_8_0_0 = tuple(int(s) for s in pa.__version__.split(".") if s.isnumeric()) < (
+    8,
+    0,
+    0,
+)
+pytestmark = pytest.mark.skipif(
+    PYARROW_LE_8_0_0, reason="hudi only supported if pyarrow >= 8.0.0"
+)
+
+
+def _extract_testing_table(target_path) -> str:
+    fixture_path = "../examples/data/hudi-tables/0.x_cow_partitioned.zip"
```

Comment:
i think you can use this path from the root python directory to fix the CI test failure: `ray/data/examples/data/hudi-tables/0.x_cow_partitioned.zip`. or alternatively i think this also works (but prefer the full path if it works): `example://hudi-tables/0.x_cow_partitioned.zip`

Reply:
resolved!

Reply:
have to use `_resolve_paths_and_filesystem` to work with `example://`